lib/swaps/Swaps.ts

```diff
@@ -187,6 +243,14 @@ class Swaps extends EventEmitter {
 
     this.addDeal(deal);
 
+    // verify LND setup. Make sure we are connected to BTC and LTC and that
+    // the peer is also connected to these networks.
+    const err = this.verifyLNDSetup(deal, peer);
```

Comment:
I think it's a bit clear to to rename this to `errMsg` or similar to clarify that this is a message and not an "error" in the sense of a NodeJS `Error` or one of our internal error objects with a corresponding code.